### ISSUE TITLE
Progress bar on collection page

### ISSUE THREAD

**Issue by matsbov** [AtlasOfLivingAustralia/collectory#251]
On the Records tab of the collection page there is a progress bar showing how many of the total/estimated number of specimens in the collection are available in the LA portal. However, if the total number of specimens is not available it defaults to displaying the message "No records are available for viewing..." which is often wrong and produces a very confusing page where the left part says there are a lot of records available and the right part says no records:

![Screenshot-2](https://github.com/user-attachments/assets/c49dd641-1400-4200-978e-a5f4d4330b86)

This PR (https://github.com/AtlasOfLivingAustralia/collectory/pull/250) fixes this by simply hiding the progress bar if the total/estimated number of specimens is not available since the progress bar makes no sense at all in that case.

**Comment by adam-collins** [comment]
This is a data entry issue.

This section of the Records tab shows the difference between records provided to the ALA vs the records in the collection. 

To fix this, enter the number of records that the collection has into the correct field.
1. Edit the collection.
2. Find the "Edit" button beneath the title "Number of specimens in the collection".
3. Enter the number of records in the collection into the "Number of specimens" field.
4. Save the changes and check the Records tab of the collection.

For the given example I manually entered the number of records as the "Number of specimens" to confirm that this works.

**Comment by matsbov** [comment]
Yeah, I am aware the progress bar works fine if there is data available. It's just that out of the 78 collections in our Collectory we have two where the number of specimens have been entered. We will never have that data for all collections. Just randomly clicking around the collections in ALA and some of the other LivingAtlases it's not uncommon to see this. So I still think it's better to hide the progress bar if the number of specimens have not been entered.

**Comment by patkyn** [comment]
While the number of specimens need to be populated in order to see the progress bar for eg: https://collections.ala.org.au/public/show/co118, however, this information may not always be supplied by the data provider which we need to make awareness of this. 
The message "No records are available for viewing in the Atlas of Living Australia" is rather misleading as it does not even indicate that the number of specimen is missing. I do agree that it does make sense to hide the progress bar in the UI perspective, however I think the wording need to be changed as well to show why the progress bar is not shown, something like "Specimen record count is not available to show the progress bar for records".

**Comment by adam-collins** [comment]
I'll merge the PR and add this alternate text.

**Comment by peggynewman** [comment]
Is it a UI thing to give a reason why something is not showing? Don't you just not show it at all?

**Comment by adam-collins** [comment]
Applying @patkyn's input the results in the following
- < 100% progress bar, https://collections-test.ala.org.au/public/show/co119 
- 100% progress bar, https://collections-test.ala.org.au/public/show/co118
- Missing metadata, https://collections-test.ala.org.au/public/show/co116

**Comment by adam-collins** [comment]
Test links are no longer working as the changes are approved and now in a PR.
